<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1094</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="tables">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="view_btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>View</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>500</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="plot_widget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>731</width>
     <height>521</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="all_data">
    <attribute name="title">
     <string>All</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="reactor_data">
    <attribute name="title">
     <string>Reactor</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="vapor_data">
    <attribute name="title">
     <string>Vapor</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QCheckBox" name="filter_checkbox">
   <property name="geometry">
    <rect>
     <x>253</x>
     <y>70</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter data</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
